server/models/handlers/appointment.js

```diff
@@ -48,8 +48,39 @@ async function cancelWorkerAppointments(workerId, specificDate) {
     }
 }
 
+async function cancelSpecificAppointment(appointmentId) {
+    // If this method is somehow called without specifying values for the required parameters, false is returned.
+    if (_.isNil(appointmentId)) return false;
+    if (!_.isNumber(appointmentId)) return false; 
+
+    try {
+        //Check if appt exists
+        const doesAppointmentExist = await appointmentModel.appointmentExists(appointmentId);
+        if (doesAppointmentExist == false) return false;
+
+        //low priority, check if appt is not cancelled 
+
+        // Cancels specific appointment.
+        const cancelSpecificAppointment = await appointmentModel.cancelSpecificAppointment(appointmentId);
+        //if (_.isEmpty(cancelSpecificAppointment)) return false;
```

Comment:
Hi Peter, 
Not sure if this is temporarily in the code for testing purposes, but I would recommend either removing it OR providing a brief explanation as to why it is kept in the codebase.

Reply:
Removed!